dodo.py

```diff
@@ -258,7 +259,7 @@ class U:
     @staticmethod
     def do(*args, cwd=P.ROOT, **kwargs):
         """wrap a CmdAction for consistency"""
-        return doit.tools.CmdAction(list(args), shell=False, cwd=str(Path(cwd)))
+        return doit.tools.CmdAction(list(args), shell=platform.system() == 'Windows', cwd=str(Path(cwd)))
```

Comment:
Thanks for the PR!

See more thoughts on #70, but assuming all the shell tokens are escaped is usually bad for business once there's any kind of weirdness in strings (spaces, non-ascii, etc). Posted an alternate, painfully-won approach there...

I am not entirely sure if we can take on supporting windows as a fully-supported platform at this stage, and maybe ever if we get into bespoke emscripten stuff for pyolite, but if we do, it will have to involve a CI excursion.